docs/status.md

```diff
@@ -53,7 +53,7 @@ one of the following tracking labels.
 | compare                  |         no         |        yes*        |        yes*         |         yes          |        no        |
 | complex                  |         no         |        yes*        |        yes*         |         yes          |        no        |
 | compute_reshape_shape    |         no         |         no         |         no          |         yes          |        no        |
-| concatenate              |         no         |        yes*        |        yes*         |         yes          |        no        |
+| concatenate              |        yes         |        yes*        |        yes*         |         yes          |        no        |
```

Comment:
Can we check our spec with verifier and shape inferencer?

Reply:
I checked that it follows the spec except one constraint. Currently, verify does not check that size of `inputs` is at least one:
* (C3) `inputs` have N tensors where N >= 1.

Reply:
If HLO support concatenate with zero args, then maybe we should also support that? 

However it seems that the verifier does check that
https://github.com/openxla/stablehlo/blob/4859634f3d95851a7a55c16cf9d753327a57c860/stablehlo/dialect/StablehloOps.cpp#L3022

Also the HLO semantics.

Reply:
Ah I didn't catch that part. Yes, then concatenate is in sync.